baseband/mark4/header.py

```diff
@@ -391,9 +419,12 @@ def fromvalues(cls, ntrack, decade=None, **kwargs):
         ----------
         ntrack : int
             Number of Mark 4 bitstreams.
-        decade : int, optional
-            Decade the observations were taken (needed to remove ambiguity in
-            the Mark 4 time stamp; not needed if a ``time`` is given).
+        decade : int, or None, optional
+            Decade in which the observations were taken.  Can instead pass an
```

Comment:
Here, we should keep the sentence from above stating that this is only needed if one does not give an explicit `time`.